pkg/codegen/docs/gen_kubernetes.go

```diff
@@ -60,6 +62,24 @@ func getKubernetesOverlayPythonFormalParams(modName string) []formalParam {
 				DefaultValue: "=None",
 			},
 		}
+	case "kustomize":
```

Comment:
Just curious about this. Do we have consistency in the ordering of the formal params between all languages of this new component?

Reply:
No, the `resourcePrefix` and `transformations` parameters are not in the same order across all SDKs.